### DISCUSSION TITLE
Integration Documentation for Paperless - Redirect URL change in current version

### DISCUSSION THREAD

**Discussion by m-georgi** [authelia/authelia#7735] [Documentation Feedback] [answered]
I was setting up an integration with Paperless yesterday following the [integration documentation](https://www.authelia.com/integration/openid-connect/paperless/).

I am using Paperless in Version [2.11.4](https://github.com/paperless-ngx/paperless-ngx/releases/tag/v2.11.4), the documentation is for Version [2.7.2](https://github.com/paperless-ngx/paperless-ngx/releases/tag/v2.7.2).

I was able to set up the integration, but I had to change the `redirect_uris` parameter.

The one in the documentation is: `https://paperless.example.com/accounts/authelia/login/callback/`
The one that was working for me is: `https://paperless.example.com/accounts/oidc/authelia/login/callback/`

I have not checked whether this is due to the difference in my deployed version of Paperless, i.e. whether it would have worked with the stated version in the documentation.

 What is your policy regarding the `Tested Versions` section in your integration documentation?
If you want, I can open a pull request to reflect the necessary change for a working setup with the more up-to-date version.

**Comment by james-d-elliott** [comment]
> What is your policy regarding the `Tested Versions` section in your integration documentation?
> If you want, I can open a pull request to reflect the necessary change for a working setup with the more up-to-date version.

We'd love a PR updating it. We have no formal policy but we'd prefer you update the versions when you test certain combinations. These docs are community driven and we only intend to guide people on new versions of software where applicable.

**Reply by james-d-elliott** [reply]
Also to answer your question, I probably introduced it with the latest version information solely based on their docs at the time. It's likely the cause of the inconsistency is their docs being incorrect or a simple mistake on my part.

**Reply by hendrik1120** [reply]
It was changed by their django-allauth dependency a while back. A few other django applications might be affected as well.

**Reply by m-georgi** [reply, marked as answer]
Thank you for your feedback!

I opened PR #7756 to update the docs

**Comment by cromelex** [comment]
@hendrik1120 thanks for mentioning this, I was getting the same error.
Can confirm this is the only changed, working with paperless 2.11.6
